### PR TITLE
change base consensus from Ostracon to Tendermint v0.34.24

### PR DESCRIPTION
**Pull request by zemyblue**
## Description

Change base consensus from Ostracon to Tendermint. 
The base codebase of Ostracon v1.1.2 is tendermint v0.34.24. 
So I convert to tendermint v0.34.24.

### Changes
* Change Ostracon abci to tendermint abci.
* Remove async mempool feature.
  * accountwgs.go, baseapp/reactor.go
  * This feature will be treated by Finschia/cometbft#1.
* Remove rpc IdleTimeout setting (Finschia/finschia-sdk#114)
  * This feature will be treated by Finschia/cometbft#38.
* Change those importing Ostracon to refer to tendermint.
* Remove Ostracon API protos.
  * proto/lbm/base/ostracon/v1/query.proto
  * proto/lbm/tx/v1beta1/service.proto
  * third_party/proto/ostracon/*
* Fixed bug where log level was not set through CLI.

We already fixed #821 and cosmos-sdk also need to fix it.

related: #1177 

## Checklist:

- [ ] I followed the [contributing guidelines](https://github.com/Finschia/finschia-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/finschia-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Comment by ulbqb** [comment]
What is the codebase version?

**Comment by zemyblue** [comment]
> What is the codebase version?

 The codebase of Ostracon v1.1.2 is Tendermint v0.34.24.
So I did convert to Tendermint v0.34.24.

**Review by ulbqb — DISMISSED** [review]

File: client/query.go [hunk 1]
```diff
@@ -7,12 +7,11 @@ import (
 
 	"github.com/pkg/errors"
 	abci "github.com/tendermint/tendermint/abci/types"
+	tmbyptes "github.com/tendermint/tendermint/libs/bytes"
```
Comment (ulbqb) [inline client/query.go:10]:
```suggestion
	tmbytes "github.com/tendermint/tendermint/libs/bytes"
```

File: client/query.go [hunk 2]
```diff
@@ -45,7 +44,7 @@ func (ctx Context) QueryWithData(path string, data []byte) ([]byte, int64, error
 // QueryStore performs a query to a Tendermint node with the provided key and
 // store name. It returns the result and height of the query upon success
 // or an error if the query fails.
-func (ctx Context) QueryStore(key ostbytes.HexBytes, storeName string) ([]byte, int64, error) {
+func (ctx Context) QueryStore(key tmbyptes.HexBytes, storeName string) ([]byte, int64, error) {
```
Comment (ulbqb) [inline client/query.go:47]:
```suggestion
func (ctx Context) QueryStore(key tmbytes.HexBytes, storeName string) ([]byte, int64, error) {
```

File: client/query.go [hunk 3]
```diff
@@ -124,7 +123,7 @@ func sdkErrorToGRPCError(resp abci.ResponseQuery) error {
 // query performs a query to a Tendermint node with the provided store name
 // and path. It returns the result and height of the query upon success
 // or an error if the query fails.
-func (ctx Context) query(path string, key ostbytes.HexBytes) ([]byte, int64, error) {
+func (ctx Context) query(path string, key tmbyptes.HexBytes) ([]byte, int64, error) {
```
Comment (ulbqb) [inline client/query.go:126]:
```suggestion
func (ctx Context) query(path string, key tmbytes.HexBytes) ([]byte, int64, error) {
```

File: client/query.go [hunk 4]
```diff
@@ -140,7 +139,7 @@ func (ctx Context) query(path string, key ostbytes.HexBytes) ([]byte, int64, err
 // queryStore performs a query to a Tendermint node with the provided a store
 // name and path. It returns the result and height of the query upon success
 // or an error if the query fails.
-func (ctx Context) queryStore(key ostbytes.HexBytes, storeName, endPath string) ([]byte, int64, error) {
+func (ctx Context) queryStore(key tmbyptes.HexBytes, storeName, endPath string) ([]byte, int64, error) {
```
Comment (ulbqb) [inline client/query.go:142]:
```suggestion
func (ctx Context) queryStore(key tmbytes.HexBytes, storeName, endPath string) ([]byte, int64, error) {
```

**Comment by ulbqb** [comment]
The word `ostracon` looks to be left in `.golangci.yml`, `Makefile`.
The words  `ocNode`, `ocPk`, `ocProtoPk`, `ostPrivKey`, `ostPubKey`, `ostos`, and `ostcrypto/etc` looks to be left in go files.

**Review by ulbqb — COMMENTED** [review]
LGTM

**Review by ulbqb — APPROVED** [review]

**Review by tkxkd0159 — APPROVED** [review]
LGTM

**Pull request merged**